vars/rpmDashboardsDistValidation.groovy

```diff
@@ -84,12 +84,12 @@ def call(Map args = [:]) {
         if (osd_status_json != 0) {
             sleep 10
             osd_status_json = sh (
-                    script: "curl -s \"http://localhost:5601/api/status\" -u admin:admin",
+                    script: "curl -s \"http://localhost:5601/api/status\" -u admin:myStrongPassword123!",
```

Comment:
this change should be version based. (admin for < 2.12 and myStrongPassword123! >= 2.12)